src/azure-cli/azure/cli/command_modules/eventhubs/_params.py

```diff
@@ -104,7 +104,7 @@ def load_arguments_eh(self, _):
 
     for scope in ['eventhubs eventhub update', 'eventhubs eventhub create']:
         with self.argument_context(scope) as c:
-            c.argument('message_retention_in_days', options_list=['--message-retention'], type=int, help='Number of days to retain events for this Event Hub, value should be 1 to 7 days and depends on Namespace sku. if Namespace sku is Basic than value should be one and is Manadatory parameter. Namespace sku is standard value should be 1 to 7 days, default is 7 days and is optional parameter')
+            c.argument('message_retention_in_days', options_list=['--message-retention'], type=int, help='Number of days to retain events for this Event Hub, value should be 1 to 7 days and depends on Namespace sku. if Namespace sku is Basic than value should be one and is Manadatory parameter. Namespace sku is standard value should be 1 to 7 days, default is 7 days and is optional parameter', deprecate_info=c.deprecate(hide=True, expiration='2.46.0'))
```

Comment:
We don't allow breaking changes in `2.46.0`. The next breaking change window would be May 2023. At that time we will release CLI `2.49.0`